features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/source/src/app/components/Applications/Details/InfoBar.js

```diff
@@ -0,0 +1,306 @@
+            return Promise.all([response, promised_tier]);
+        }).then(
+            response => {
+                let [application, tier] = response.map(data => data.obj);
+                this.setState({ application, tierDescription: tier.description });
+            }).catch(
+                error => {
+                    if (process.env.NODE_ENV !== "production") {
+                        console.log(error);
+                    }
+                    let status = error.status;
+                    if (status === 404) {
+                        this.setState({notFound: true});
+                    }
+                }
+            );
+}
+
+  render() {
+    const { classes, theme } = this.props;
+    const application = this.state.application;
+    if (this.state.notFound) {
+      return <ResourceNotFound message={this.props.resourceNotFountMessage} />;
+    }
+    if (!application) {
+      return <Loading />;
+    }
+    return (
+      <div className={classes.infoBarMain}>
+        <div className={classes.root}>
+          <Link to="/applications" className={classes.backLink}>
+            <KeyboardArrowLeft className={classes.backIcon} />
+            <div className={classes.backText}>
+              BACK TO <br />
+              LISTING
+            </div>
+          </Link>
+          <div className="vertical-divider-70" />
+          <div style={{ marginLeft: theme.spacing.unit }}>
+            <Typography variant="display1">sddff</Typography>
+            <Typography variant="caption" gutterBottom align="left">
+              {application.name} | 21-May 2018
+            </Typography>
+          </div>
+        </div>
+
+        {this.state.showOverview && 
+        <Collapse in={this.state.showOverview} timeout="auto" unmountOnExit>
+          <div className={classes.infoContent}>
+            <div className={classes.contentWrapper}>
+              <div className={classes.topBar}>
+                <div className={classes.infoItem}>
+                  <Typography variant="subheading" gutterBottom>
+                    {application.throttlingTier} <Typography variant="caption">( {this.state.tierDescription} )</Typography>
+                  </Typography>
+                  <Typography variant="caption" gutterBottom align="left">
+                    Throttling Tier 
+                  </Typography>
+                </div>
+                <div className={classes.infoItem}>
+                  {application.lifeCycleStatus === "APPROVED" ?
+                      <CheckCircle /> 
+                      :
+                      <Typography variant="subheading" gutterBottom>{this.state.lifeCycleStatus}</Typography> 
+                  }
+                  <Typography variant="caption" gutterBottom align="left">
+                    lifeCycleStatus
+                  </Typography>
+                </div>
+
+              </div>
+              <Typography>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut
```

Comment:
What does this mean to be? Looks like it's non English.

Reply:
This is just dummy text to fill the UI and provide the next developer who is going to implement and plugin the description part. Lorem ipsum is a popular site to generate dummy text and it's widely use by designers.